dkg-test-suite/src/testVAnchorConfigurableLimitsProposal.ts

```diff
@@ -0,0 +1,241 @@
+import { ApiPromise } from '@polkadot/api';
+import { Keyring } from '@polkadot/keyring';
+import { hexToBytes, listenOneBlock, provider, waitNfinalizedBlocks } from './utils';
+import { ethers } from 'ethers';
+import { keccak256 } from '@ethersproject/keccak256';
+import { ECPair } from 'ecpair';
+import { assert } from '@polkadot/util';
+
+const raw_data =
+	'00000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001';
+const tokenUpdateProp = new Uint8Array(hexToBytes(raw_data));
+
+async function testDirectProposal() {
+	const api = await ApiPromise.create({
+		provider,
+		types: {
```

Comment:
Can we move these types into a file and export them for these tests? Also these types should be updated in the other tests now that this PR introduces new types.

Reply:
Alright